decentraland/unity-renderer
# style: backpack refinement

## What does this PR change?
Architectural and visual improvements in the sections side.
- Sections and subsetions buttons moved upward.
- Categories section width is now smaller.
- Items and spaces between cells are smaller.
- Warning message in ItemsToggle prefab is updated.
- Loading items animation is fixed.

## How to test the changes?
Test link: https://play.decentraland.zone/?renderer-branch=style/BackpackRefinement

Press [I] to open the backpack menu and navigate all the sections to check everything is consistent and that loading items animations are working.
